.github/workflows/release-docker-github.yml

```
@@ -6,10 +6,11 @@ name: Docker Release to Github
 # documentation.
 
 on:
-  workflow_dispatch:
-  push:
-    tags:
-      - "v*"
+  workflow_call:
+    outputs:
+      VERSION:
+        description: release version
+        value: ${{ jobs.build-and-push-image.outputs.VERSION }}
```

actionlint (1.7.4)

property "build-and-push-image" is not defined in object type {build: {outputs: {version: string}}}

(expression)